plugins/core-classes/src/main/java/org/qbicc/plugin/coreclasses/CoreClasses.java

```diff
@@ -41,6 +43,10 @@
         builder.setEnclosingType(enclosing);
         return builder.build();
     };
+    private static final String OBJECT_INT_NAME = "java/lang/Object";
```

Comment:
Instead of making these private static final, should we have a central list of constants for the internal name of well-known classes?  We've got class names as strings all over the place...

Reply:
Yeah, that's probably a good idea, but I'll save that for a follow up because I'm weary of conflicts :)